### PR TITLE
Add helm improvements

### PR DESCRIPTION
**Pull request by grafjo**
this helps using a dedicated (private) image registry -  e.g. building own images

### CODE REVIEW

**Comment by grafjo** [comment]
@sbckr DCO bot is missing, is that the expected behavior?

File: helmfile.d/0100.castor.yaml [hunk 1]
```diff
@@ -24,7 +24,14 @@ releases:
...
       image:
-        tag: 0.1-SNAPSHOT-1576642733-7-72de814
+        registry: {{ env "CASTOR_IMAGE_REGISTRY" | default "ghcr.io" }}
+        repository: {{ env "CASTOR_IMAGE_REPOSITORY" | default "carbynestack/castor-service" }}
+        tag: {{ env "CASTOR_IMAGE_TAG" | default "0.1-SNAPSHOT-1576642733-7-72de814" }}
+        {{ if env "IMAGE_PULL_SECRET" }}
+        pullSecrets:
+          - {{ env "IMAGE_PULL_SECRET" }}
+        {{ end }}
+
```
Comment (strieflin) [inline helmfile.d/0100.castor.yaml:34]: In order to follow a consistent formatting strategy, please remove all intermittent new lines (this one, same file below and in other files).

File: helmfile.d/0200.amphora.yaml [hunk 2]
```diff
@@ -31,7 +31,13 @@ releases:
       vcPartner: {{ requiredEnv "AMPHORA_VC_PARTNER_URI" }}
       noSslValidation: {{ env "NO_SSL_VALIDATION" | default false }}
       image:
-        tag: 0.1-SNAPSHOT-1576737535-6-7798b9e
+        registry: {{ env "AMPHORA_IMAGE_REGISTRY" | default "ghcr.io" }}
+        repository: {{ env "AMPHORA_IMAGE_REPOSITORY" | default "carbynestack/amphora-service" }}
+        tag: {{ env "AMPHORA_IMAGE_TAG" | default "latest" }}
```
Comment (strieflin) [inline helmfile.d/0200.amphora.yaml:36]: We would like to have deterministic deployments. Please avoid using `latest`. Instead please use fixed version numbers as has been the case before.

File: helmfile.d/0300.ephemeral.yaml [hunk 3]
```diff
@@ -23,16 +23,29 @@ releases:
   values:
     - discovery:
         image:
-          tag: 0.1-SNAPSHOT-1525401638-9-a0b5e7a
+          registry: {{ env "DISCOVERY_IMAGE_REGISTRY" | default "ghcr.io" }}
+          repository: {{ env "DISCOVERY_IMAGE_REPOSITORY" | default "carbynestack/ephemeral/discovery" }}
+          tag: {{ env "DISCOVERY_IMAGE_TAG" | default "latest" }}
```
Comment (strieflin) [inline helmfile.d/0300.ephemeral.yaml:28]: Pin down version (see comment above).

**Review by strieflin — CHANGES_REQUESTED** [review]

**Comment by grafjo** [comment]
@strieflin thx for your review - fixed all issues

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**